system/lib/wasmfs/streams.cpp

```diff
@@ -34,7 +34,7 @@ std::shared_ptr<StdinFile> StdinFile::getSingleton() {
 }
 
 __wasi_errno_t StdoutFile::write(const uint8_t* buf, size_t len, off_t offset) {
-  return writeStdBuffer(buf, len, &emscripten_console_log, writeBuffer);
+  return writeStdBuffer(buf, len, &emscripten_out, writeBuffer);
```

Comment:
please add a link to the issue with node that is the reason for using out/err here.